cmd/openshift/operator/kodata/tekton-addon/1.4.0/addons/02-clustertasks/buildah/buildah-v0-22-0-task.yaml

```diff
@@ -1,15 +1,15 @@
 # auto generated by script/update-tasks.sh
 # DO NOT EDIT: use the script instead
-# source: https://raw.githubusercontent.com/openshift/tektoncd-catalog/release-v0.19/task/buildah/0.1/buildah.yaml
+# source: https://raw.githubusercontent.com/openshift/tektoncd-catalog/release-v0.22/task/buildah/0.1/buildah.yaml
 #
 ---
 ---
 apiVersion: tekton.dev/v1beta1
 kind: ClusterTask
 metadata:
-  name: buildah-v0-19-0
+  name: buildah-v0-22-0
   labels:
-    app.kubernetes.io/version: "1.4.0"
+    app.kubernetes.io/version: "0.1"
```

Comment:
@piyush-garg why `0.1` :thinking:

Reply:
@savitaashture it looks like this version is the Task version (the version directory in catalog)
the 1.4.0 comes up in the versioned taskname. This is useful as this will help us traceback to the right version of the task in catalog later.

This make me think that we need an additional label to indicate operator release version. so that a user/dev can identify which release delivered a clusterTask (or whether an upgrade has happened or not)

@vdemeester